pkg/k8s/slim/k8s/apis/core/v1/types.go

```diff
@@ -120,6 +120,64 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type ConditionStatus string
+
+// These are valid condition statuses. "ConditionTrue" means a resource is in the condition.
+// "ConditionFalse" means a resource is not in the condition. "ConditionUnknown" means kubernetes
+// can't decide if a resource is in the condition or not. In the future, we could add other
+// intermediate conditions, e.g. ConditionDegraded.
+const (
+        ConditionTrue    ConditionStatus = "True"
+        ConditionFalse   ConditionStatus = "False"
+        ConditionUnknown ConditionStatus = "Unknown"
+)
```

Comment:
This needs to be between lines 99 and 100

Reply:
will do, but curious why?

Reply:
@Weil0ng just to make it in the same order as the upstream version. It is easier to do a diff between both files if things are kept in the same order.

Reply:
I see, thanks!

Reply:
and done :)